devicetypes/smartthings/nyce-motion-sensor.src/nyce-motion-sensor.groovy

```diff
@@ -166,66 +166,21 @@ private Map parseIasMessage(String description) {
 def refresh()
 {
 	log.debug "refresh called"
-	[
-		"st rattr 0x${device.deviceNetworkId} 1 1 0x20"
-
-	]
+	return zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020) //battery read
 }
 
 def ping() {
 	refresh()
 }
 
 def configure() {
-
-	String zigbeeEui = swapEndianHex(device.hub.zigbeeEui)
 	log.debug "Configuring Reporting, IAS CIE, and Bindings."
-	def configCmds = [
-		"zcl global write 0x500 0x10 0xf0 {${zigbeeEui}}", "delay 200",
```

Comment:
Isn't this configuring the CIE address? Is it just unused by the device?

Reply:
That is done as a part of the `enrollResponse` call as well.